ConTabs/Column.cs

```diff
@@ -7,14 +7,45 @@
 namespace ConTabs
 {
     [DebuggerDisplay("Column for '{PropertyName}'")]
+
+	/// <summary>
+	/// Acts as a column within a table
+	/// </summary>
     public class Column
     {
+		/// <summary>
+		/// The defined type of the given property
+		/// </summary>
         public Type SourceType { get; set; }
+
+		/// <summary>
+		/// The name of the given property
+		/// </summary>
         public string PropertyName { get; private set; }
+
+		/// <summary>
```

Comment:
Worth mentioning this can be overwritten as desired? And is used for the column heading.